functions/src/verifier.ts

```diff
@@ -77,24 +55,49 @@ const genAuthProof = async (uid: string, identity: string,
       KMS_KEY_TYPE[KMS_KEY_TYPE.operator],
       toEthSignedMessageHash(message)
   ) as string;
-  const validatorAddr = kmsConfig().get(
-      KMS_KEY_TYPE[KMS_KEY_TYPE.operator]
-  )?.publicAddress;
   const encodedSig = ethers.utils.defaultAbiCoder.encode(
-      ["address", "bytes"], [validatorAddr, sig]
+      ["uint256", "address", "bytes"],
+      [expiredAt, validator, sig]
   );
 
   const AuthProof: AuthProof = {
-    name: nameHash,
+    name: nameHashRes.nameHash!,
     requestId: requestId,
-    issuedAt: issuedAt,
-    identityType: identity,
-    authType: auth,
-    signature: encodedSig,
+    expiredAt: expiredAt,
+    validator: validator,
+    proof: encodedSig,
   };
   return {code: 200, authProof: AuthProof};
```

Comment:
maybe we can simply return the encodedSig here

Reply:
makes sense.